webview-ui/src/Kaito/KaitoModels.tsx

```diff
@@ -115,19 +122,21 @@ resource:
       apps: ${appname}
 inference:
   preset:
-    name: ${inferencename}`;
-        return yaml;
+    name: ${inferencename}${privateConfig}`;
+        return [yaml, gpu];
     }
 
     function generateCRD(model: string) {
-        const yaml = generateYAML(model);
+        const yaml = generateYAML(model)[0];
         vscode2.postGenerateCRDRequest({ model: yaml });
         return;
     }
 
     function onClickDeployKaito(model: string) {
-        const yaml = generateYAML(model);
-        vscode2.postDeployKaitoRequest({ model: model, yaml: yaml });
+        const [yaml, gpu] = generateYAML(model);
+        if (!(gpu === undefined)) {
+            vscode2.postDeployKaitoRequest({ model: model, yaml: yaml, gpu: gpu });
```

Comment:
💡 Can we please avoid `numeric` like `vscode2` value if that is not the intent for contextual understanding? (We can come out with more understandable and human readable variable names here :) 

Let definitely refactor this sooner please. 

things like this should have better names:

```*.ts
export const vscode2 = getWebviewMessageContext<"kaitoModels">({
    generateCRDRequest: null,
    deployKaitoRequest: null,
    workspaceExistsRequest: null,
    updateStateRequest: null,
    resetStateRequest: null,
    cancelRequest: null,
});
```